index.html

```diff
@@ -360,6 +360,37 @@ <h2>Skills</h2>
             </ul>
         </div>
     </section>
+    <!-- End Section part -->
+
+    <!-- Start Contact form -->
+
+    <div class="contact-form">
```

Comment:
Kindly use flexbox to positioning the elements in contact form. Currently, you are using `display:block`. In the very next lessons, you will start to code desktop version of the portfolio and you need to give responsiveness with the Flexbox. Please check this [article](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) to learn more about Flexbox.